cmd/ufs/ufs.go

```diff
@@ -1,13 +1,21 @@
-// This is a ufs server.
+// This is a userspace server which exports a filesystem over 9p(2000).
+//
+// UFS exports a filesystem from the server running it via 9p over
+// a network connection.
+//
+// By default, it will export / over a TCP on port 5640 under the username
+// of "harvey".
+//
+// UFS is unrelated to the UFS filesystem from BSD.
```

Comment:
I took it out, but I had included it because I was confused by the name and I figured other people not familiar with the project might be too. 

(For what it's worth, I just checked and the sidebar on wikipedia claims UFS originated with 4.2BSD, but even if not it's most strongly affiliated with the BSDs these days.)